docs/additional-features/chainbridge/definitions.md

```diff
@@ -0,0 +1,188 @@
+$ cb-sol-cli bridge register-resource \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --resourceId "[RESOURCE_ID]" \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[HANDLER_CONTRACT_ADDRESS]" \
+  --targetContract "[TARGET_CONTRACT_ADDRESS]"
+```
+
+### Make contract burnable/mintable
+
+```bash
+# Let contract burnable/mintable
+$ cb-sol-cli bridge set-burn \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[HANDLER_CONTRACT_ADDRESS]" \
+  --tokenContract "[TARGET_CONTRACT_ADDRESS]"
+```
+
+### Cancel proposal
+
+Cancel proposal for execution
+
+```bash
+# Cancel ongoing proposal
+$ cb-sol-cli bridge cancel-proposal \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --resourceId "[RESOURCE_ID]" \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --chainId "[CHAIN_ID_OF_SOURCE_CHAIN]" \
+  --depositNonce "[NONCE]"
+```
+
+### Pause/Unpause
+
+Pause deposits, proposal creation, voting, and deposit executions temporally.
+
+```bash
+# Pause
+$ cb-sol-cli admin pause \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]"
+
+# Unpause
+$ cb-sol-cli admin unpause \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]"
+```
+
+### Change Fee
+
+Change the fee which will be paid to Bridge Contract
+
+```bash
+# Change fee for execution
+$ cb-sol-cli admin set-fee \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --fee [FEE_IN_WEI]
+```
+
+### Add/Remove a relayer
+
+Add an account as a new relayer or remove an account from relayers
+
+```bash
+# Add relayer
+$ cb-sol-cli admin add-relayer \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --relayer "[NEW_RELAYER_ADDRESS]"
+  
+# Remove relayer
+$ cb-sol-cli admin remove-relayer \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --relayer "[RELAYER_ADDRESS]"
+```
+
+### Change relayer threshold
+
+Change the number of votes required for a proposal execution
+
+```bash
+# Remove relayer
+$ cb-sol-cli admin set-threshold \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --threshold [THRESHOLD]
+```
+:::
+
+## Chain ID
+
+The Chainbridge `chainId` is an arbitrary value used in the bridge for differentiating between the blockchain networks. To not be confused with the chain ID of the network, they are not the same thing. This value needs to be unique (not the same as the chain ID of the network), since two networks might have the same one.
+
+In this example, we set  `99` in `chainId`, because the chain ID of the Mumbai testnet is `80001`, which cannot be represented with a uint8.
```

Comment:
Please explain chainID is uint8 value before this example. It makes people confused a little bit.

```
chainID is must be in the range of uint8.
...

For example, we use `99` for Mumbai chain whose network ID is `80001` in this guide because the networkID is not in the range of uint8.
```